lib/binarisYML.js

```diff
@@ -60,13 +49,16 @@ const getFuncName = function getFuncName(binarisConf) {
 };
 
 const checkFuncConf = async function checkFuncConf(funcConf, funcDirPath) {
-  if (!Object.prototype.hasOwnProperty.call(funcConf, fileStr)) {
-    throw new Error(`${binarisConfFile}: function missing required field: <${fileStr}>`);
+  for (const section of [fileStr, entryStr, runtimeStr]) {
+    if (typeof funcConf[section] !== 'string') {
+      throw new Error(`${binarisConfFile}: field: <${fileStr}> should be a string`);
+    }
   }
-  if (!Object.prototype.hasOwnProperty.call(funcConf, entryStr)) {
-    throw new Error(`${binarisConfFile}: function missing required field: <${entryStr}>`);
+  const fullPath = path.join(funcDirPath, funcConf[fileStr]);
+  const stats = await fs.stat(fullPath);
+  if (!stats.isFile()) {
+    throw new Error(`Not a file in path: ${fullPath}`);
```

Comment:
This error is user-visible? If so, it should be more informative.

Reply:
I just brought back the code that attempts to read the function and throws the generic fs.readFile error